pkg/tenant/tenant_test.go

```diff
@@ -86,6 +87,7 @@ func (suite *tenantTestSuite) SetupSuite() {
 			Testing: true,
 		},
 		Ensign: config.EnsignConfig{
+			Endpoint: "bufconn",
```

Comment:
Turns out this was the problem all along; can't pass an empty string to the new version of the `grpc` connector. See: https://github.com/grpc/grpc-go/issues/2838. I think we bumped some subdependencies which is what caused this error to trigger.